cmd/osm-controller-bootstrap/osm-controller-bootstrap_test.go

```diff
@@ -55,7 +55,8 @@ func TestCreateDefaultMeshConfig(t *testing.T) {
 	"enableValidatingWebhook": false,
 	"enableIngressBackendPolicy": true,
 	"enableEnvoyActiveHealthChecks": true,
-	"enableSnapshotCacheMode": true
+	"enableSnapshotCacheMode": true,
+	"enableCRDConverter": true
```

Comment:
Let's always enable this? It's no longer optional since MeshConfig and other bootstrap functionality is moved into this pod.

Reply:
I actually kept this to determine if we want to patch the CRDs with the conversion webhook. But I guess its not needed and we can just turn it on by default